definitions/npm/mongoose_v4.x.x/flow_v0.50.x-/mongoose_v4.x.x.js

```diff
@@ -492,15 +495,15 @@ declare class Mongoose$Connection {
 }
 
 declare module "mongoose" {
-  declare export type MongooseConnection = Mongoose$Connection;
-  declare export type MongoId = MongoId;
-  declare export type BSONObjectId = bson$ObjectId;
-  declare export type ObjectId = bson$ObjectId;
-  declare export type MongooseQuery<Result, Doc> = Mongoose$Query<Result, Doc>;
-  declare export type MongooseDocument = Mongoose$Document;
-  declare export type MongooseModel = typeof Mongoose$Document;
-  declare export type MongooseSchema<Doc> = Mongoose$Schema<Doc>;
-  declare export type MongooseSchemaField<Schema> = Mongoose$SchemaField<
+  declare type MongooseConnection = Mongoose$Connection;
+  declare type MongoId = MongoId;
+  declare type BSONObjectId = bson$ObjectId;
+  declare type ObjectId = bson$ObjectId;
+  declare type MongooseQuery<Result, Doc> = Mongoose$Query<Result, Doc>;
+  declare type MongooseDocument = Mongoose$Document;
+  declare type MongooseModel = typeof Mongoose$Document;
+  declare type MongooseSchema<Doc> = Mongoose$Schema<Doc>;
+  declare type MongooseSchemaField<Schema> = Mongoose$SchemaField<
```

Comment:
Explain more about this decision. By removing these types from exports, this would potentially break consumers. Why is it necessary to stop exporting them?